From c03c1ba88d9b319513667dbab12a78dfdc3006ea Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Thu, 7 Dec 2017 10:42:54 +0200
Subject: [PATCH 19/48] ARM: dts: imx7d: sbc-imx7: add support for baseboard
 eeprom

Add support for SB-SOM-iMX7 baseboard EEPROM over I2C4 bus.

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 arch/arm/boot/dts/imx7d-sbc-imx7.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sbc-imx7.dts b/arch/arm/boot/dts/imx7d-sbc-imx7.dts
index 7dd1055f..3ca3e4e 100644
--- a/arch/arm/boot/dts/imx7d-sbc-imx7.dts
+++ b/arch/arm/boot/dts/imx7d-sbc-imx7.dts
@@ -37,6 +37,12 @@
 		#gpio-cells = <2>;
 		reg = <0x20>;
 	};
+
+	eeprom_sb@54 {
+		compatible = "atmel,24c08";
+		reg = <0x54>;
+		pagesize = <16>;
+	};
 };
 
 &pwm2 {
-- 
2.7.4

